environment/bashrc/.bashrc_vm

```diff
@@ -0,0 +1,22 @@
+module purge
```

Comment:
* Is this `purge` command really required?  This is usually a bad idea as the developer might have orthogonal modules loaded (emacs, htop, etc.) that would unexpectedly be removed from the environment.
* If this isn't required, then you don't need to define the specialized `dracoenv` bash function in my requested replacement of this file.